### PR TITLE
merge the sprint2016 page that is already linked from mails and twitter

### PR DESCRIPTION
**Pull request by hpk42**
forgot to do a PR, thanks Florian.

### CODE REVIEW

File: doc/en/announce/sprint2016.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+--------------
+
+Here are preliminary participants who said they are likely to come,
+given some expenses funding::
+
+    Andreas Pelme, Sweden
+    Brianna Laugher, Australia
+    Bruno Oliveira, Brazil
+    Floris Bruhin, Switzerland
+    Florian Bruynooghe, UK
```
Comment (The-Compiler) [inline doc/en/announce/sprint2016.rst]: before someone merges this I'm going to point out this is wrong here again :laughing:

File: tox.ini [hunk 2]
```diff
@@ -93,7 +93,6 @@ deps=sphinx
      PyYAML
 
 commands=
-    {envpython} plugins_index/plugins_index.py
```
Comment (The-Compiler) [inline tox.ini]: @nicoddemus I suggested removing this when @hpk42 had trouble building the docs - I think it's okay to remove it as the plugin index got replaced by plugincompat on Heroku anyways, correct?

**Comment by hpk42** [comment]
let me do this again some other time.

File: tox.ini (hunk 2 above)
Comment (nicoddemus) [inline tox.ini]: Yep, that can be removed

**Comment by nicoddemus** [comment]
Hi guys,

Florian is right, we decided the plugin index was not very useful anymore
so that script was removed. Sorry I missed that.

I fixed that directly in master, as the fix is to just delete that line so
I think it did not warrant a PR.

Thanks,
Bruno.

**Pull request closed without merging**